e2e-tests/security-context/compare/job.batch_restore-job-restore-pvc-sec-context-80.yml

```diff
@@ -0,0 +1,72 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    job-name: restore-job-restore-pvc-sec-context
+  name: restore-job-restore-pvc-sec-context
+  ownerReferences:
+  - apiVersion: pxc.percona.com/v1
+    controller: true
+    kind: PerconaXtraDBClusterRestore
+    name: restore-pvc
+spec:
+  backoffLimit: 4
+  completions: 1
+  parallelism: 1
+  selector:
+    matchLabels:
+  template:
+    metadata:
+      labels:
+        job-name: restore-job-restore-pvc-sec-context
+    spec:
+      containers:
+      - command:
+        - recovery-pvc-joiner.sh
+        env:
+        - name: RESTORE_SRC_SERVICE
+          value: restore-src-restore-pvc-sec-context
+        - name: XB_USE_MEMORY
+          value: "750000000"
```

Comment:
hm, why operator counts different values for 8.0 and 5.7?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/337/commits/7846fba987d6e9c36ef1589103c3e4df513068b8
Probably when I run tests I did not have latest version from master branch.